### PR TITLE
User access control - Attributes

### PR DESCRIPTION
**Pull request by jamesm-macrometa**
This PR adds GUI and C8QL documentation for the Attributes feature.

### CODE REVIEW

**Comment by jamesm-macrometa** [comment]
GUI tutorials are close to being ready for review, but the C8QL tutorials might be better off in a separate PR.

**Comment by jamesm-macrometa** [comment]
@diana-macrometa FYI, this is a little unclear in its current state without a tutorial on how to integrate attributes into queries.  My next priority will be to add the query part as a separate PR.

File: docs/account-management/permissions/attributes/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 10
+title: Attributes
+---
+
+Administrators can assign attributes to users and API keys which limit document access. You can use attributes to assign read or write permissions at the document level or collection level.
```
Comment (diana-macrometa) [inline docs/account-management/permissions/attributes/index.md]:
```suggestion
Administrators can assign attributes to users and API keys which limit document access. You can use attributes to assign read or write permissions at the document level or collection level. For more information about permissions, refer to [Permissions](LINK TO PERMISSIONS DEFINITION TOPIC).
```

File: docs/account-management/permissions/attributes/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+In this example, imagine a fulfillment company with documents and collections that must be accessed by partners and employees. The partners request items for the fulfillment company to provide, and the company employees fulfill the orders of the partners' items.
```
Comment (diana-macrometa) [inline docs/account-management/permissions/attributes/index.md:8]: This is great.

File: docs/account-management/permissions/attributes/remove-attributes-api.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 50
+title: Remove Attributes from API Keys
+---
+
+To remove attributes from an API key:
```
Comment (diana-macrometa) [inline docs/account-management/permissions/attributes/remove-attributes-api.md:6]: We should crosslink to this from the API keys section.

**Review by diana-macrometa — APPROVED** [review]
Great stuff. A couple minor suggestions.

**Comment by diana-macrometa** [comment]
Not sure what you mean about C8QL tutorials.

I suggest that you hold off on doing more until we see what's going on with using attributes, which should be in the not-distant future. When that happens, we can actually draft a coherent tutorial or example based on that usage.

**Pull request merged**